### DISCUSSION TITLE
Qwen2.5-72B-Instruct-GPTQ-Int4 可以lora微调并合并权重吗？

### DISCUSSION THREAD

**Discussion by YingxuanW** [QwenLM/Qwen3#1035] [Q&A]
请问可以lora微调Qwen2.5-72B-Instruct-GPTQ-Int4模型吗？如果可以，lora权重要如何合并？

**Comment by jklj077** [comment]
不能哈。倒是可以试试和Qwen2.5-72B-Instruct合并后再量化。
